### PR TITLE
Fe material web

### PR DESCRIPTION
**Pull request by lorericci**
I provided a visual update of the web application

### CODE REVIEW

File: client/src/components/Map.tsx [hunk 1]
```diff
@@ -93,11 +95,13 @@ const MapComponent: FC<MapComponentProps> = (props) => {
...
-                    alt="Custom Marker"
-                    style={{ width: "4vh", height: "4vh" }}
-                  />
+                  {
+                    <img
+                      src={`/document-${doc.type}-icon.png`}
+                      alt="Custom Marker"
+                      style={{ width: "4vh", height: "4vh" }}
+                    />
+                  }
```
Comment (michepaolo) [inline client/src/components/Map.tsx:104]: delete parenthesis,

File: client/src/components/NavHeader.tsx [hunk 2]
```diff
@@ -22,15 +22,19 @@ const NavHeader: FC<NavHeaderProps> = ({
     <nav className="nav-header">
       <div className="nav-container">
         <div className="nav-left">
-          <img
+          {/*<img
             src="/LOGO.png"
             alt="Kiruna eXplorer Logo"
             className="nav-logo"
-          ></img>
+          ></img>*/}
```
Comment (michepaolo) [inline client/src/components/NavHeader.tsx:29]: Replace logo

File: client/src/global.d.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+declare namespace JSX {
+  interface IntrinsicElements {
+    "md-filled-tonal-icon-button": React.DetailedHTMLProps<
+      React.HTMLAttributes<HTMLElement>,
+      HTMLElement
+    >;
+    "md-icon": React.DetailedHTMLProps<
+      React.HTMLAttributes<HTMLElement>,
+      HTMLElement
+    >;
```
Comment (michepaolo) [inline client/src/global.d.ts:10]: Delete this part

**Review by michepaolo — CHANGES_REQUESTED** [review]

File: client/src/components/Map.tsx (hunk 1 above)
Comment (umberto-fontanazza) [reply client/src/components/Map.tsx:104]: Don't put styles in the components. Put that in the scss file

**Comment by lorericci** [comment]
Now the PR is ready to be merged

**Review by michepaolo — APPROVED** [review]

**Pull request merged**